azure-pipelines.yml

```
@@ -69,14 +69,13 @@ stages:
             steps:
               - checkout: self
                 clean: true
-              - powershell: |
-                  SqlLocalDB stop
-                  SqlLocalDB delete
-                  Remove-Item "$env:USERPROFILE\*.mdf"
-                  Remove-Item "$env:USERPROFILE\*.ldf"
-                  SqlLocalDB create
-                  SqlLocalDB start
-                displayName: Cleanup databases
+              - powershell: >
+                  Get-ChildItem -ErrorAction SilentlyContinue -Recurse 'C:\Program Files\Microsoft SQL Server\sqlcmd.exe' |
+                  Sort-Object -Property @{Expression={$_.FullName.Length}; Descending=$true},@{Expression={$_.FullName}; Descending=$true} |
+                  Select-Object -First 1 |
+                  Foreach-Object {"##vso[task.prependpath]$($_.DirectoryName)"}
```

Still hitting the error about Microsoft. Is it as simple as&hellip;?
```suggestion
                  Foreach-Object {Write-Host "##vso[task.prependpath]$($_.DirectoryName)"}
```

---

I don't think it's related as it shows up even without the powershell script